### ISSUE TITLE
Compile instruction improvement

### ISSUE THREAD

**Issue by qubodup** [sergiomb2/pinball.old#1]
Arch Linux 64bit.

Hello, I'd love to try out this repository's emilia pinball, as it seems to be the most up-to-date. Unfortunately I can't compile it.
1. User/group instructions

```
NOTE: A user and group named 'games' must exist before installation, this
user and group exist by defualt on most systems else:
useradd 'games' (as root user)
```

Is this true? I used --prefix because I hate installing as root ( http://stackoverflow.com/questions/3239343/make-install-but-not-to-default-directories ) and it worked, so perhaps that avoids it. I'm pretty sure I don't have a 'game' user
1. Steps to compile

```
configure
make 
make install (as root user)
make clean (if you want to save some disc space)
```

I'm guessing the first line should be `./configure.sh`. It works in the 0.3.2 release. In current git as well, but it shows a warning for about 10 seconds, which made me think at the first try that it's frozen (my bad):

```
aclocal: warning: couldn't open directory 'libltdl/m4': No such file or directory
```

I suggest you update the readme line to `./configure.sh`. Even though the file './configure' does appear after running ./configure.sh...
1. Multithread make fails

`make -j7` fails in both git and 0.3.2 but running `make -j1` compiles successfully in both.

```
make[2]: Leaving directory '/home/qubodup/pinball-master/src'
Makefile:548: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/qubodup/pinball-master'
Makefile:422: recipe for target 'all' failed
make: *** [all] Error 2
```

Not sure whether this requires action. Perhaps add a FAQ or Known Issues section?

**Comment by sergiomb2** [comment]
Hi, 
I update README , you don't need user games because /var/games/pinball is not in use ... but if make install stops, please tell me (Fedora has user games) 

yeah autoreconf -i before ./configure (or ./configure.sh) and just make , is more one issue to fix ... 

Thanks.

BTW:  Fedora also package this pinball version [1] you could take a look for take ideas , for example in install section :
  # remove unused global higescorefiles:
  rm -r $RPM_BUILD_ROOT%{_localstatedir}
  # remove unused test module
  rm $RPM_BUILD_ROOT%{_libdir}/%{name}/libModuleTest.*
  # .la files are needed for ltdl
  rm $RPM_BUILD_ROOT%{_libdir}/%{name}/lib*.{a,so}
  # remove bogus development files
  rm $RPM_BUILD_ROOT%{_bindir}/%{name}-config
  rm -r $RPM_BUILD_ROOT%{_includedir}/%{name}

[1] https://pkgs.fedoraproject.org/cgit/pinball.git/tree/pinball.spec

**Comment by sergiomb2** [comment]
./bootstrap
./configure
make
make install (as root ) or src/pinball
make clean (if you want to save some disc space)

yes make -j3 still fail to build

**Comment by sergiomb2** [comment]
make -j3  fixed in tag  0.3.3.1 , thanks for your help .
